tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_training_cuda11_3

```diff
@@ -183,6 +183,7 @@ CMD ["/bin/bash"]
 #Build manylinux2014 docker image end
 ARG PYTHON_VERSION=3.6
 ARG TORCH_VERSION=1.10.0
+ARG OPSET_VERSION=14
```

Comment:
Why do we need to explicitly set this?

Reply:
In that case, we don't need but I made this parameter explicit even if 14 is the default value. I did it to make it easier when we update default opset to 15.